staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -113,32 +113,32 @@ type ThirdPartyResource struct {
 	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	// Description is the description of this object.
+	// description of this object.
```

Comment:
Shouldn't these fragments be sentences?  "The description..." ?

Reply:
That's a great question @thockin - looking through this package, I saw both "The description of..." and "description of". I favored the latter because it seemed more inline with the already existing docs. Very happy to update to the former though if you think that'd be preferable :)